mdoc/Mono.Documentation/Updater/DocUtils.cs

```diff
@@ -200,6 +200,13 @@ public static bool IsDelegate (TypeDefinition type)
                     baseRef.FullName == "System.MulticastDelegate";
         }
 
+        public static bool NeedOverwrite(XmlElement element)

```

Comment:
Very minor, but please change the name of this member to `NeedsOverwrite` (note the `s`) ... it makes the code easier to read, grammatically speaking.